console/src/components/Common/Layout/ReusableHeader/types.ts

```diff
@@ -0,0 +1,9 @@
+export type HeaderState = {
```

Comment:
It's used in one place, let's keep types close to places where they are used. There's no need to create a separate file for a small type.